twentytwentyone-blocks/experimental-theme.json

```diff
@@ -0,0 +1,203 @@
+{
+	"global": {
+		"settings": {
+			"color": {
+				"gradients": [ ],
+				"palette": [
+					{
+						"slug": "black",
+						"color": "#000000"
+					},
+					{
+						"slug": "dark-gray",
+						"color": "#28303D"
+					},
+					{
+						"slug": "gray",
+						"color": "#39414D"
+					},
+					{
+						"slug": "green",
+						"color": "#D1E4DD"
+					},
+					{
+						"slug": "blue",
+						"color": "#D1DFE4"
+					},
+					{
+						"slug": "purple",
+						"color": "#D1D1E4"
+					},
+					{
+						"slug": "red",
+						"color": "#E4D1D1"
+					},
+					{
+						"slug": "orange",
+						"color": "#E4DAD1"
+					},
+					{
+						"slug": "yellow",
+						"color": "#EEEADD"
+					},
+					{
+						"slug": "white",
+						"color": "#FFFFFF"
+					}
+				]
+			},
+			"typography": {
+				"customFontSize": true,
+				"customLineHeight": true,
+				"fontSizes": [
+					{
+						"slug": "extra-small",
+						"size": "16px"
+					},
+					{
+						"slug": "small",
+						"size": "18px"
+					},
+					{
+						"slug": "normal",
+						"size": "20px"
+					},
+					{
+						"slug": "large",
+						"size": "24px"
+					},
+					{
+						"slug": "extra-large",
+						"size": "40px"
+					},
+					{
+						"slug": "huge",
+						"size": "96px"
+					},
+					{
+						"slug": "gigantic",
+						"size": "144px"
+					}
+				],
+				"spacing": {
+					"customPadding": true,
+					"units": [ "px", "em", "rem", "vh", "vw" ]
+				}
+			},
+			"custom": {
+				"font-primary": "-apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen-Sans, Ubuntu, Cantarell, 'Helvetica Neue', sans-serif",
+				"font-weight-light": 300,
+				"color": {
+					"primary": "var(--wp--preset--color--dark-gray)",
+					"secondary": "var(--wp--preset--color--gray)",
+					"primary-hover": "var(--wp--custom--color-primary)",
+					"background": "var(--wp--preset--color--green)",
+					"border": "var(--wp--custom--color--primary)"
+				},
+				"line-height": {
+					"body": 1.7,
+					"heading": 1.3,
+					"page-title": 1.1
+				},
+				"responsive": {
+					"aligndefault-width": "610px",
+					"alignwide-width": "1240px"
+				},
+				"spacing": {
+					"unit": "20px",
+					"horizontal": "25px",
+					"vertical": "30px"
+				}
+			}
+		},
+		"styles": {
+	    	"color": {
+				"background": "var(--wp--custom--color--background)",
+				"text": "var(--wp--custom--color--primary)",
+				"link": "var(--wp--custom--color--primary)"
+			},
+			"typography": {
+				"fontSize": "var(--wp--preset--font-size--normal)",
+				"lineHeight": "var(--wp--custom--line-height--body)"
+			}
+		}
+	},
+	"core/paragraph": {
+		"styles": {
+			"color": {
+				"text": "var(--wp--custom--color--primary)",
```

Comment:
Would applying "text": "var(--wp--custom--color--primary)", at the global level instead of for each text block also work?

Reply:
I believe the intention is to prepare the different colors this way in order to help developers or advanced users to change the values?
Similar to how the CSS variables can be used in the PHP version of the theme.

Reply:
> Would applying "text": "var(--wp--custom--color--primary)", at the global level instead of for each text block also work?

It should — I believe when I tried it, it didn't work though so I did this instead. I'd have to dig back into it and double check.